weasyl/controllers/user.py

```diff
@@ -92,19 +92,20 @@ def GET(self):
                 "month": None,
                 "year": None,
                 "error": None,
+                "display_reCaptcha": True,
             },
         ])
 
     @define.token_checked
     def POST(self):
         form = web.input(
             username="", password="", passcheck="", email="", emailcheck="",
-            day="", month="", year="", recaptcha_challenge_field="", recaptcha_response_field="")
+            day="", month="", year="", recaptcha_challenge_field="", g_recaptcha_response=web.input("g-recaptcha-response"))
```

Comment:
This is a bit.. confused. I can't remember the best way to pull out fields with `-` instead of `_`, but `web.input` in `web.input` is not great.